test/v5.spec.js

```diff
@@ -6,19 +6,19 @@ const Vue = require('vue/dist/vue.js')
 const VueRx = require('../dist/vue-rx.js')
 
 // library
-const Observable = require('rxjs/Observable').Observable
-const Subject = require('rxjs/Subject').Subject
-const Subscription = require('rxjs/Subscription').Subscription
-require('rxjs/add/observable/fromEvent')
-require('rxjs/add/operator/share')
+const Observable = require('rxjs-compat/Observable').Observable
```

Comment:
I would import from the proper locations expected in v6... other `rxjs` or `rxjs/operators`. Not from `rxjs-compat`. That should never be imported from directly anyhow.